src/test/java/com/microsoft/sqlserver/jdbc/callablestatement/CallableMixedTest.java

```diff
@@ -0,0 +1,103 @@
+package com.microsoft.sqlserver.jdbc.callablestatement;
+
+import java.sql.*;
+
+import com.microsoft.sqlserver.jdbc.RandomUtil;
+import com.microsoft.sqlserver.jdbc.SQLServerDriver;
+import com.microsoft.sqlserver.jdbc.TestUtils;
+import com.microsoft.sqlserver.testframework.AbstractSQLGenerator;
+
+import org.junit.jupiter.api.Test;
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.runner.RunWith;
+import static org.junit.jupiter.api.Assertions.fail;
+
+
+@RunWith(JUnitPlatform.class)
+public class CallableMixedTest {
+
+    @Test
+    public void datatypestest() throws Exception {
+        Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");

```

Comment:
I dont think we need these lines to load that class into classpath for our tests